<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editmandialog</class>
 <widget class="QDialog" name="editmandialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maneuver Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maneuver Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mantype">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select maneuver type and click Apply button</string>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>64</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date&amp;Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="isotedit">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>61</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Date and Time (ISOT format)</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>44</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ISOT : yyyy-mm-ddThh:mm:ss.ssssss</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jdedit">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>61</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Date and Time (Julian day)</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>45</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>JD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>93</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="parameters">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>521</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="finishbutton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Finish editing</string>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelbutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Discard changes and quit editing</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showorbit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show orbit</string>
   </property>
   <property name="text">
    <string>SHOW Orbit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="applymantype">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>9</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select maneuver type and click Apply button</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mantypedisp">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="computeFTA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Comute fixed time arrival guidance</string>
   </property>
   <property name="text">
    <string>FTA</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_exec">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Finish editing and execute this maneuver</string>
   </property>
   <property name="text">
    <string>Finish and Exec</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="duration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>61</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Flight Duration for FLYTO</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EditDateTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_duration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>44</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (days)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="optimize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Optimize Assistant</string>
   </property>
   <property name="text">
    <string>OPTIMIZE</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mantype</tabstop>
  <tabstop>applymantype</tabstop>
  <tabstop>isotedit</tabstop>
  <tabstop>jdedit</tabstop>
  <tabstop>parameters</tabstop>
  <tabstop>showorbit</tabstop>
  <tabstop>computeFTA</tabstop>
  <tabstop>finish_exec</tabstop>
  <tabstop>finishbutton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
